docs/source/tutorials/odsc_tutorial/sagemaker_backend.rst

```diff
@@ -0,0 +1,100 @@
+
+   python benchmarking/nursery/odsc_tutorial/transformer_wikitext2/sagemaker/launch_remote.py \
+     --experiment_tag tmlr-10 --benchmark transformer_wikitext2 \
+     --random_seed 2938702734 --scale_max_wallclock_time 1 \
+     --num_seeds 5 --n_workers <n-workers>
+
+Here, ``<n_workers>`` is 2, 4, 8 respectively.
+
+* We run 5 random repetitions (seeds), therefore 20 experiments per value of
+  ``<n_workers>``.
+* Running the experiments for ``<n_workers>`` requires a resource limit larger
+  or equal to ``<n_workers> * 20`` for instance type ``ml.g4dn.xlarge``. If
+  your limit is less than this, you should launch fewer experiments in
+  parallel, since otherwise most of the experiments will not be able to use
+  ``<n_workers>`` workers.
+* With ``--scale_max_wallclock_time 1``, we adjust ``max_wallclock_time`` if
+  ``n_workers`` is smaller than the default value (4) for our benchmark. In
+  our example, the case ``--n_workers 2`` runs for 10 hours instead of 5.
+
+Once all experiments are finished, with results written to S3, we can create
+a plot comparing the performance across different numbers of workers, using
+the following script:
+
+.. literalinclude:: ../../../../benchmarking/nursery/odsc_tutorial/transformer_wikitext2/sagemaker/plot_results.py
+   :caption: transformer_wikitext2/sagemaker/plot_results.py
+   :start-after: # permissions and limitations under the License.
+
+For details about visualization of results in Syne Tune, please consider
+`this tutorial <../visualization/README.html>`__. In a nutshell:
+
+* Different to the plot above, we have four subplots here, one for each method.
+  In each subplot, we compare results for different numbers of workers.
+* ``metadata_to_subplot`` configures grouping w.r.t. subplot (depends on
+  method), while ``metadata_to_setup`` configures grouping w.r.t. each curve
+  shown in each subplot (depends on ``n_workers``).
+
+Here is the plot:
+
+.. |SageMaker transformer_wikitext2| image:: img/odsc-comparison-sagemaker-transformer_wikitext2.png
+
++-----------------------------------------------------------+
+| |SageMaker transformer_wikitext2|                         |
++===========================================================+
+| Comparison of methods on transformer_wikitext2 benchmark, |
+| using the SageMaker backend with 2, 4, 8 workers.         |
++-----------------------------------------------------------+
+
+* In general, we obtain good results faster with more workers. However,
+  especially for ``BO`` and ``MOBSTER``, the improvements are less pronounced
+  than one might expect.
+* Our results counter a common misconception, that as we go to higher degrees of
+  parallelization of trials, the internals of the HPO method do not matter
+  anymore, and one might as well use random search. This is certainly not the
+  case for our problem, where ``BO`` with 2 workers attains a better
+  performance after 5 hours than ``RS`` with 8 workers, at a quarter of the
+  cost.
```

Comment:
Great result and a good thing to highlight. 

Minor nit: the x-axis units are seconds, but they might be more readable if they were converted to hours. 

![Screen Shot 2023-06-02 at 5 15 49 PM](https://github.com/awslabs/syne-tune/assets/1182047/9faf59c7-0bdf-4d7b-8418-d91d26b4a600)

Reply:
That would be another feature for the plotting code.